gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -6,7 +6,7 @@ option java_package = "io.camunda.zeebe.gateway.protocol";
 option go_package = "./;pb";
 
 // For a more complete documentation, refer to Zeebe documentation at:
-// https://docs.camunda.io/docs/reference/grpc
+// https://https://docs.camunda.io/docs/apis-tools/zeebe-api/overview/
```

Comment:
```suggestion
// https://docs.camunda.io/docs/apis-tools/zeebe-api/overview/
```

I guess this was not intended :)

Reply:
Not sure where the doc was pointing to before, but somehow I feel it would make sense to point to https://docs.camunda.io/docs/apis-tools/zeebe-api/gateway-service/ or maybe collide the overview and gateway-service 🤔 the overview page looks so empty 🤷🏼

Reply:
Updated!